sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -190,87 +258,39 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
         """.stripMargin)
 
       // LOAD DATA INTO partitioned table must specify partition
-      intercept[AnalysisException] {
-        sql(s"""LOAD DATA LOCAL INPATH "$testData" INTO TABLE part_table""")
+      withInputFile { path =>
+        intercept[AnalysisException] {
+          sql(s"""$loadQuery INPATH "$path" INTO TABLE part_table""")
+        }
+
+        intercept[AnalysisException] {
```

Comment:
The error message is wrong. 
`LOAD DATA target table `default`.`part_table` is partitioned, but number of columns in provided partition spec (1) do not match number of partitioned columns in table (s2);`

`s2` is incorrect. We need to remove `s` from the following code: 
```
s"(s${targetTable.partitionColumnNames.size})"
```